diff --git a/commerce_cart_expiration.module b/commerce_cart_expiration.module
index 6eeede4..6476952 100644
--- a/commerce_cart_expiration.module
+++ b/commerce_cart_expiration.module
@@ -291,8 +291,10 @@ function commerce_cart_expiration_page_explanation() {
   }
 
   // Otherwhise create content here without replacing commerce_order tokens.
-  $explanation_content = variable_get('commerce_cart_expiration_explanation_page', '<p>'. array('value' => t('Sorry, you took too much time in the checkout process.') .'</p><p>'. t('<a href="[site:url]">Return to the front page.</a>') .'</p>', 'format' => NULL));
-  $text = check_markup($explanation_content['value'], $explanation_content['format']);
+  $text = variable_get('commerce_cart_expiration_explanation_page', '<p>'. t('Sorry, you took too much time in the checkout process.') .'</p><p>'. t('<a href="[site:url]">Return to the front page.</a>') .'</p>');
+  if (is_array($text)) {
+    $text = check_markup($text['value'], $text['format']);
+  }
 
   return token_replace($text, array('site' => NULL, 'user' => $user));
 }
